java/src/jmri/jmrit/display/layoutEditor/LayoutEditor.java

```diff
@@ -1161,7 +1158,7 @@ private void createFloatingEditToolBox() {
 
             if (isEditable() && floatingEditToolBox == null) {
                 //Create the window and add the toolbox content
-                floatingEditToolBox = new JmriJFrame("ToolBox: " + layoutName); // TODO - Bundle
+                floatingEditToolBox = new JmriJFrame(rb.getString("ToolBox") + ": " + layoutName);
```

Comment:
This should be ```Bundle.getMessage("ToolBox", layoutName)``` instead of ```rb.getString(...)```
(you can replace all uses of ```rb.getString(...)``` with ```Bundle.getMessage(...)```).